llama_index/llms/openai_utils.py

```diff
@@ -107,29 +110,45 @@
 CompletionClientType = Union[Type[Completion], Type[ChatCompletion]]
 
 
-def _create_retry_decorator(max_retries: int) -> Callable[[Any], Any]:
+def create_retry_decorator(
+    max_retries: int,
+    exponential: bool = True,
+    stop_after_delay_seconds: Union[int, None] = None,
+) -> Callable[[Any], Any]:
     min_seconds = 4
     max_seconds = 10
-    # Wait 2^x * 1 second between each retry starting with
-    # 4 seconds, then up to 10 seconds, then 10 seconds afterwards
+    wait_strategy = (
+        wait_exponential(multiplier=1, min=min_seconds, max=max_seconds)
+        if exponential
```

Comment:
Imo this arg name is misleading, because regardless of setting `exponential` as true or false, an exponential gets used.  Would you mind updating it to be `random_exponential`?  Also, why not pass `min_seconds` and `max_seconds` to `wait_random_exponential`?

Reply:
1)Yep might be better to have a more accurate name 
2) Can also probably keep max_seconds and min_seconds as arguments the util function with certain default values?